scripts/gcs/flask_gcs.py

```diff
@@ -8,221 +8,241 @@
-        self.img_proc_start_time = self.manager.Value('i', int(datetime.utcnow().strftime("%s")))
-        self.targets_to_submit = self.manager.list()
-
-        self.interop_client = self.manager.list()
-        self.interop_client.append(SUASSystem.InteropClientConverter())
-        self.interop_data = self.manager.list()
-        self.interop_data.append(self.get_interop_data())
-
-        self.gcs_process = multiprocessing.Process(target=SUASSystem.gcs_process, args=(
-            self.sda_status,
-            self.img_proc_status,
-            self.interop_client,
-            self.targets_to_submit
-        ))
-        self.gcs_process.start()
-
-    def get_interop_data(self):
-        try:
-            active_interop_mission = self.interop_client[0].get_active_mission()
-            obstacles = self.interop_client[0].get_obstacles()
-            active_interop_mission_json = SUASSystem.get_mission_json(active_interop_mission, obstacles)
-
-            data = {
-                "status": "connected",
-                "emergent_position": [
-                    active_interop_mission_json["emergent_last_known_pos"]["latitude"],
-                    active_interop_mission_json["emergent_last_known_pos"]["longitude"]
-                ],
-                "airdrop_position": [
-                    active_interop_mission_json["air_drop_pos"]["latitude"],
-                    active_interop_mission_json["air_drop_pos"]["longitude"]
-                ],
-                "off-axis_position": [
-                    active_interop_mission_json["off_axis_target_pos"]["latitude"],
-                    active_interop_mission_json["off_axis_target_pos"]["longitude"]
-                ]
-            }
-        except:
-            data = {
-                "status": "disconnected",
-                "emergent_position": [0, 0],
-                "airdrop_position": [0, 0],
-                "off-axis_position": [0, 0]
-            }
-
-        return data
-
-    def set_sda_status(self, status):
-        if status == "connected":
-            self.sda_start_time.value = int(datetime.utcnow().strftime("%s"))
-
-        self.sda_status.value = status
-
-    def get_sda_status(self):
-        return self.sda_status.value
-
-    def set_img_proc_status(self, status):
-        if status == "connected":
-            self.img_proc_start_time.value = int(datetime.utcnow().strftime("%s"))
-
-        self.img_proc_status.value = status
-
-    def get_img_proc_status(self):
-        return self.img_proc_status.value
+	def __init__(self):
+		self.manager = multiprocessing.Manager()
+		self.sda_status = self.manager.Value('s', "disconnected")
+		self.img_proc_status = self.manager.Value('s', "disconnected")
+		self.sda_start_time = self.manager.Value('i', int(datetime.utcnow().strftime("%s")))
+		self.img_proc_start_time = self.manager.Value('i', int(datetime.utcnow().strftime("%s")))
+		self.targets_to_submit = self.manager.list()
+
```

Comment:
Don't put a blank line here.

Reply:
Blank line removed in https://github.com/FlintHill/SUAS-Competition/commit/2cfe923afe9cd34b3304a8761765fe817b092ac9